sources/packages/api/src/utilities/address-utils.ts

```diff
@@ -0,0 +1,84 @@
+import { AddressDetailsAPIOutDTO } from "../route-controllers/models/common.dto";
+import { AddressDetailsModel } from "../services";
+import { AddressInfo } from "../types";
+
+/**
+ * Util to transform address details.
+ * @param addressDetails is a partial AddressDetailsModel.
+ * @returns address info as AddressInfo.
+ */
+export function transformAddressDetails(
+  addressDetails: AddressDetailsModel,
+): AddressInfo {
+  return {
+    addressLine1: addressDetails.addressLine1,
+    addressLine2: addressDetails.addressLine2,
+    province: addressDetails.provinceState,
+    country: addressDetails.country,
+    city: addressDetails.city,
+    postalCode: addressDetails.postalCode,
+    selectedCountry: addressDetails.selectedCountry,
+  };
+}
+/**
+ * Util to transform address details for formIO.
+ * @param addressDetails is a partial AddressDetailsModel.
+ * @returns address info as AddressDetailsAPIOutDTO.
+ */
+export function transformAddressDetailsForForm(
```

Comment:
It seems that this class belongs to the DTO scope only.
The utils are supposed to be something that falls under a helper that will be common and can be used in any layer.

Reply:
mm, are you suggesting to move it to the dto file?

Reply:
I would say yes because it is specifically doing something for DTOs.

Reply:
okay. I will update this in my future PR